app/src/main/res/values/floats.xml

```diff
@@ -1,4 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <item name="horizontal_visible_item_count" format="float" type="dimen">1.3</item>
+
```

Comment:
nit: could you please add a link (maybe https://material.io/guidelines/style/typography.html ?) to explain where these values are from. 😄

Reply:
Good point! Thanks @jmatsu

Reply:
@jmatsu Thanks for your advice!

The link is:
* https://material.io/guidelines/style/typography.html#typography-line-height

The values comes like below:

* For Example, "Headline" of "Dense and tall scripts" has been designated "Type 24sp, Leading 34dp",  so I thought about using android:lineSpacingExtra with 10dp ( = 34dp -24sp on normal font setting) first.

* However, If we use this approach with larger font setting, line spacing will be relatively narrower, so I changed to using android:lineSpacingMultiplier with 1.42 (≒ 34dp ÷ 24sp on normal font setting) in the case.

* And, "Title"  has been designated "should exist as single lines". so I think we don't have any designated leading value,  but It may become multi-line on this app, so I checked  Zeplin Scene and adapted manually.

Is it better if I write comments about these values on xml file?

Or.. using android:lineSpacingExtra with SP (not dp) might be more simle and better.  How do you think?

Reply:
yes! please comment it on the xml. 😍

Reply:
`android:lineSpacingExtra` doesn't depend on a text size which a TextView has.
So, I guess `android:lineSpacingMultiplier` is better in this case 🤔 
@konifar what do you think?